.gitignore

```diff
@@ -1,3 +1,9 @@
+# Generated by Cargo
+# will have compiled files and executables
 /target
 /ptau
-/benches/csv
\ No newline at end of file
+/benches/csv
+
+# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
```

Comment:
I'd say it doesn't hurt to have it checked in into Git to prevent any potential issues. Do you imagine the crate being used as a dependency by someone in the future?

Reply:
I still don't know how it will be consumed. But I agree with you that it is better to stay on the safe side for now. Removing it from `.gitignore`